### PR TITLE
Add missed doc-comments for validation result (T430255)

### PR DESCRIPTION
**Pull request by lukyanovas**

### CODE REVIEW

**Review by DokaRus — APPROVED** [review]
LGTM

**Review by TikhonovaTatyana — APPROVED** [review]

File: js/ui/validation_engine.js [hunk 1]
```diff
@@ -441,9 +445,29 @@ var GroupConfig = Class.inherit({
...
+             * @publicName brokenRules
+             * @type Array<dxValidationRule>
+             */
             brokenRules: [],
+            /**
+             * @name dxValidationGroupResult_validators
+             * @publicName validators
+             * @type Array<Object>
+             */
             validators: []
```
Comment (kaatula) [inline js/ui/validation_engine.js:471]: That is also strictly typed interface (or even just array of dxValidators, can't say for sure)

File: ts/dx.all.d.ts [hunk 2]
```diff
@@ -6326,18 +6338,34 @@ declare module DevExpress.viz {
...
+    /** An object that defines a gauge indicator of the rangeBar type. */
+    export interface linearRangeBar extends CommonIndicator {
+        /** Specifies the distance between the indicator and the invisible scale line. */
+        offset?: number;
+    }
     /** An object that defines a gauge indicator of the triangleMarker type. */
     export interface linearTriangleMarker extends CommonIndicator {
         /** Specifies the indicator length. */
         length?: number;
+        /** Specifies the distance between the indicator and the invisible scale line. */
```
Comment (kaatula) [inline ts/dx.all.d.ts:6367]: What's that? Not referenced in PR's title

File: js/ui/validation_engine.js (hunk 1 above)
Comment (lukyanovas) [reply js/ui/validation_engine.js:471]: Yes, thank you. I'll add this types а bit later. Unfortunately, it's not just an array of dxValidator.

File: ts/dx.all.d.ts (hunk 2 above)
Comment (lukyanovas) [reply ts/dx.all.d.ts:6367]: It is a side effect of the experiment with TS in charts sources. Yes, I agree that it should be part of another PR.

**Pull request merged**